person_capture/gui_app.py

```
@@ -13950,6 +14041,18 @@ def _write_sdr_crop_from_bgr(
             crop_bgr = np.ascontiguousarray(bgr[y1:y2, x1:x2, :3], dtype=np.uint8)
             if crop_bgr.size <= 0:
                 return False, "empty_sdr_crop"
+            smooth_changed = 0
+            try:
+                crop_bgr, smooth_changed = self._smooth_wic_shadow_noise_bgr(crop_bgr)
+            except Exception as smooth_exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)